app/views/foreman_leapp/job_templates/leapp_preupgrade.erb

```diff
@@ -6,12 +6,20 @@ description_format: 'Upgradeability check for RHEL host'
 provider_type: SSH
 feature: leapp_preupgrade
 model: JobTemplate
+template_inputs:
+- name: Channel
+  description: Set if the target OS is required to use any Extended Upgrade Support (EUS), Advanced Update Support (AUS), Update Services for SAP Solutions (E4S) repository or none.
+  input_type: user
+  required: true
+  default: "none"
+  options: "none\neus\naus\ne4s"
 %>
 
 <%= render_template 'Check Leapp' %>
 
 rm -f /var/log/leapp/leapp-report.json
-leapp preupgrade
+<% channel_opts = input('Channel') == 'None' ? '' : "--channel #{input('Channel')}" %>
```

Comment:
Condition is for `None`, but in the inputs the default value is `none`.

Reply:
simplified by adopting the same channels as `leapp`: none is gone, ga is default.